cst/plugins/eu.esdihumboldt.cst.functions.string/src/eu/esdihumboldt/cst/functions/string/DateExtraction.java

```diff
@@ -65,6 +66,14 @@ protected Object evaluate(String transformationIdentifier, TransformationEngine
 		String sourceString = variables.values().iterator().next().getValueAs(String.class);
 		SimpleDateFormat sdf = new SimpleDateFormat(dateFormat);
 
+		// by default setLenient(true)
+		boolean leniency = getOptionalParameter(PARAMETER_LENIENCY, Value.of(true))
+				.as(Boolean.class);
+
+		if (leniency == false) {
+			sdf.setLenient(false);
```

Comment:
As you have a boolean now you could also just set it to the value of `leniency` in any case.

Reply:
This is also done and tested. I merge it.